gui/functionality/remove_cluster.py

```diff
@@ -16,6 +16,10 @@ def remove_cluster(self):
         cluster_name = self.ui.cluster_name_removec_edit.text()
         self.parameters['remove_cluster_arguments']['cluster_name'] = cluster_name
         self.saver.save(self.parameters)
-        remove_cluster(cluster_name)
+        try:
+            remove_cluster(cluster_name)
+        except KeyError as e:
+            self.window = ErrorApp("Cluster not exists")
```

Comment:
Shouldn't it be better with text `Cluster does not exist`?

Reply:
Ok, changed.